README.md

```diff
@@ -78,10 +78,27 @@ Split allows you to [implement a custom impression listener](https://help.split.
 
 ## Usage
 
-Now assuming you have a split named: `feature1` you can do something like:
+Now assuming you have a split named `feature1` you can do something like:
 
-```jsx
-<Split name={'feature1'}>
+### Hook
+
+```tsx
+const [feature1, config] = useSplit('feature1');
+if (feature1 === 'on') {
+  return <Feature1 />;
+}
+```
+
+Optional [attributes](https://help.split.io/hc/en-us/articles/360020448791-JavaScript-SDK#attribute-syntax)
+can also be passed in:
+```tsx
+const [feature1, config] = useSplit('feature1', { paying_customer: true });
+```
+
+### HOC
```

Comment:
It's not a (HOC)[https://reactjs.org/docs/higher-order-components.html] it's a (render props)[https://reactjs.org/docs/render-props.html]

Reply:
My mistake. Will do!